modules/ROOT/pages/security.adoc

```diff
@@ -0,0 +1,11 @@
+= Anypoint Platform Security
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Platform provides security in several layers:
```

Comment:
I don't think the text in the bullets go with this lead-in. I suggest something like this: 

"Anypoint Platform provides several layers of security. With Anypoint Platform, you can:"